modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -222,3 +222,8 @@ To avoid this warning, replace the special character in the raw payload and then
 ----
 read(payload.^raw replace "\uFFFF" with "NonChar")
 ----
+
+== Low Performance Issues
+
+Using `output application/dw` format can impact the performance of transformations. This format is intended to help you debug the results of DataWeave transformations.
+It is significantly slower than other formats, so avoid using this format in production applications.
```

Comment:
Fine as-is but consider this:
Using `output application/dw` format helps you debug the results of DataWeave transformations. However, it is significantly slower than other formats, so avoid using this format in production applications.